python/demo/poisson/demo_poisson.py

```diff
@@ -212,9 +229,11 @@
     # If pyvista environment variable is set to off-screen (static)
     # plotting save png
     if pyvista.OFF_SCREEN:
+        # Run demo with 'PYVISTA_OFF_SCREEN=true python demo_poisson.py'
         pyvista.start_xvfb(wait=0.1)
-        plotter.screenshot("uh.png")
+        plotter.screenshot("poisson_u.png")
     else:
         plotter.show()
 except ModuleNotFoundError:
-    print("pyvista is required to visualise the solution")
+    print("'pyvista' is required to visualise the solution")
+    print("Install 'pyvista' with pip: 'python3 -m pip install pyvista'")
```

Comment:
Could the command be just `pip3 install pyvista`?

Reply:
I belive the main benefit of using `python3 -m pip` as opposed to `pip3` is that if you run the demo with `python3 demo_poisson.py` then you are sure that the dependency you install  (in the case `pyvista`) is installed in the correct environment. You can see a more thorough discussion about this topic here: https://snarky.ca/why-you-should-use-python-m-pip/